File: src/Services/NationalBank/OpenData.Services.NationalBank.API/Constants/ExchangeRatesTableType.cs
namespace OpenData.Services.NationalBank.API.Constants;

public static class ExchangeRatesTableType
{
    public static string TableA => "A";

    public static string TableB => "B";

    public static string TableC => "C";

    public static ICollection<string> All => new List<string> { TableA, TableB, TableC };
}

File: src/Clients/SPA/open-data-client/src/constants/exchangeRatesTableType.ts
const exchangeRatesTableType = () => {
  const tableA = "A";
  const tableB = "B";
  const tableC = "C";

  const all = () => [tableA, tableB, tableC];

  return { tableA, tableB, tableC, all };
};

export { exchangeRatesTableType };
